apps/server/src/modules/files-storage/controller/files-storage.controller.ts

```diff
@@ -60,4 +67,17 @@ export class FilesStorageController {
 
 		return response;
 	}
+
+	@Patch(':id/rename')
```

Comment:
By list, download, upload we have write the operations one first place and the id after it.